manticore/manticore.py

```diff
@@ -505,7 +537,189 @@ def load_assertions(self, path):
                     logger.debug("Repeated PC in assertions file %s", path)
                 self._assertions[pc] = ' '.join(line.split(' ')[1:])
 
+    def _backup_state_callback(self, state, state_id):
+        logger.debug("Backup state %r", state_id)
+
+    def _restore_state_callback(self, state, state_id):
+        logger.debug("Restore state %r", state_id)
+
+    def _terminate_state_callback(self, state, state_id, ex):
+        print "Terminate state", state.context
+
+        logger.info("Terminate_state_callback for %d", state_id)
+        executor = self._executor
+        #aggregates state statistics into exceutor statistics. FIXME split
+        logger.debug("Terminate state %r %r ", state, state_id)
+        state_visited = state.context.get('visited', set())
+        state_instructions_count = state.context.get('instructions_count', 0)
+        with self._executor.locked_context() as context:
+            executor_visited = context.get('visited', set())
+            context['visited'] = executor_visited.union(state_visited)
+
+            executor_instructions_count = context.get('instructions_count', 0)
+            context['instructions_count'] = executor_instructions_count + state_instructions_count 
+
+    def _fork_state_callback(self, state, expression, values, policy):
```

Comment:
imo, we should delete all these callbacks and their signals, if these stub callbacks are the only slots connected to them. that is, until we have a use case for them.